105875 N-K-1 InvPrint = Henry - Add a PDF image to the back of the invoice format = Henry

sachin448 opened the pull request:
Please review changes and merge into development.
File - Source/ar/rep/invhenry.p
Source/oe/rep/invhenry.p
CustFiles/Images/HenryWireInstructions.pdf

sachin448 committed:
105875 N-K-1 InvPrint = Henry - Add a PDF image to the back of the invoice format = Henry
File - Source/ar/rep/invhenry.p
Source/oe/rep/invhenry.p
CustFiles/Images/HenryWireInstructions.pdf

sachin448 committed:
105875 N-K-1 InvPrint = Henry - Add a PDF image to the back of the invoice format = Henry
File - Source/ar/rep/invhenry.p
Source/oe/rep/invhenry.p
Source/system/OutputProcs.p
CustFiles/Images/HenryWireInstructions.pdf

sachin448 committed:
105875 N-K-1 InvPrint = Henry - Add a PDF image to the back of the invoice format = Henry
File - Source/oe/rep/invhenry.p

ron-advantzware reviewed (changes requested):
Duplicate procedure "pPrintImageOnBack" in ar/rep/invhenry.p and oe/rep/invhenry.p.  Place this procedure in an include "methods/pPrintImageOnBack.i".  Replace this procedure with the common include.

sachin448 committed:
105875 N-K-1 InvPrint = Henry - Add a PDF image to the back of the invoice format = Henry
File - Source/ar/rep/invhenry.p
Source/oe/rep/invhenry.p
Source/methods/pPrintImageOnBack.i
